core/tests/test_utils.py

```diff
@@ -140,12 +158,26 @@ class TestBase(unittest.TestCase):
     }
 
     def _get_unicode_test_string(self, suffix):
+        """Tests if unicode string are working properly.
+
+        Args:
+            suffix: the suffix pattern to the UNICODE_TEST_STRING
+
+        Returns:
+            string with UNICODE_TEST_STRING suffixed with string `suffix`
```

Comment:
Also suffixed with string 'suffix' gives a feeling that we are suffixing the string `suffix` instead of the value that suffix contains. Update this to give the information in a better way.